gordo_components/builder/build_model.py

```diff
@@ -249,6 +310,7 @@ def provide_saved_model(
     output_dir: Union[os.PathLike, str],
     model_register_dir: Union[os.PathLike, str] = None,
     replace_cache=False,
+    cv_mode: str = "full",
```

Comment:
This isn't a valid option `full`

...if only there was just some structure we could use, which had such available choices predefined. :rofl: